runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionSearch.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Autocomplete, TextField, AutocompleteProps } from '@mui/material';
+import { translate } from 'src/hooks/useTranslations';
+import { Item } from '../ListGroup';
+
+export type SearchBarProps = Omit<AutocompleteProps<Item, false, false, false>, 'renderInput'> & {};
+
+const ActionSearch = ({ ...other }: SearchBarProps) => {
```

Comment:
it would be better to have just `const ActionSearch: VFC<SearchBarProps> = (props) => {`
and spread them in the component.  It is pointless to name it other if there is nothing more